easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -851,6 +851,9 @@ def test_step(self, return_output_ec=False):
                         actual_installdir = os.path.join(test_installdir, 'local')
                     else:
                         actual_installdir = test_installdir
+                    # Export the temporary installdir as an environment variable
+                    # Some tests (e.g. for astropy) require to be run in the installdir
+                    env.setvar('EB_TEST_INSTALLDIR', actual_installdir)
```

Comment:
@casparvl Maybe `EB_PYTHONPACKAGE_TEST_INSTALLDIR` is better, to signal that this is only to be used for easyconfigs that rely on the `PythonPackage` easyblock?

Reply:
Done!